docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/metatag/Tag/CgdpTemplate.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\cgov_core\Plugin\metatag\Tag;
+
+use Drupal\metatag\Plugin\metatag\Tag\MetaNameBase;
+
+/**
+ * Cgdp template.
+ *
+ * @MetatagTag(
+ *   id = "cgdp_template",
+ *   label = @Translation("Cgdp Template"),
+ *   description = @Translation("Cgdp Template Meta Properties."),
+ *   name = "cgdp.template",
+ *   group = "cgdpAdvanced_advanced",
+ *   weight = 6,
+ *   type = "label",
+ *   secure = FALSE,
+ *   multiple = FALSE
+ * )
+ */
+class CgdpTemplate extends MetaNameBase {
+  // Nothing here yet. Just a placeholder class for a plugin.
```

Comment:
I assume that you must have a class, but you don't need any implemention, and you will never need it. So can you change the comment to just "Just a placeholder class for a plugin."